lizard_auth_server/templates/lizard_auth_server/login.html

```diff
@@ -4,25 +4,27 @@
 {% block subtitle %}{% trans 'Log in' %}{% endblock %}
 
 {% block container %}
-    <div class="content">
-        {% if form.errors %}
+<div class="content">
```

Comment:
De Django docs doen het niet (of niet overal), dus dit lijkt me ook acceptabel:

https://docs.djangoproject.com/en/1.9/ref/templates/language/#templates